packages/commonwealth/client/scripts/views/modals/SessionRevalidationModal/SessionRevalidationModal.tsx

```diff
@@ -83,10 +82,18 @@ const SessionRevalidationModal = ({
                 but your wallet has the address{' '}
                 <b>{formatAddress(signedAddress!)}</b>.
               </p>
-              <p>
-                We’ve switched your active address to the one in your wallet.
-                You can switch it back in the user menu.
-              </p>
+              {signedAddressAccount ? (
+                <p>
+                  We’ve switched your active address to the one in your wallet.
+                  You can switch it back in the user menu.
+                </p>
+              ) : (
+                <p>
+                  Select <strong>Connect a new address</strong> in the user menu
+                  to connect this as a new address, or switch addresses in your
+                  wallet to continue.
+                </p>
```

Comment:
When connecting a new address during session revalidation, it works, but the page also breaks, notice the "Cannot read properties..."error

<img width="1125" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/51641047/34c847f0-023e-4728-81e2-de687605fccd">

Reply:
Tried a bunch of things but not able to reproduce this now.

cc: @raykyri